src/codegen/utilities/interpreter_helpers.py

```diff
@@ -108,6 +108,22 @@ def get_interpreter_parameter_signature(is_python_factory, params):
     return ", ".join(params_with_defaults)
 
 
+def get_instantiation_lines_for_output_params(output_parameters):
```

Comment:
is the output a parameter passed by reference? or just the return value/output. Please update the name accordingly

Reply:
updated as per suggestion